api/instance/v1/security_group_utils_test.go

```diff
@@ -62,3 +62,115 @@ func TestAPI_UpdateSecurityGroup(t *testing.T) {
 	})
 	testhelpers.Ok(t, err)
 }
+
+func TestAPI_UpdateSecurityGroupRule(t *testing.T) {
+
+	client, r, err := httprecorder.CreateRecordedScwClient("security-group-rule-test")
+	testhelpers.Ok(t, err)
+	defer func() {
+		testhelpers.Ok(t, r.Stop()) // Make sure recorder is stopped once done with it
+	}()
+
+	instanceAPI := NewAPI(client)
+
+	var (
+		zone         = scw.ZoneFrPar1
+		organization = "b3ba839a-dcf2-4b0a-ac81-fc32370052a0"
+	)
+
+	bootstrap := func(t *testing.T) (*SecurityGroup, *SecurityGroupRule, func()) {
+
```

Comment:
nit: newline

Reply:
We always start a func with a new line no ?

Reply:
[nope](https://github.com/scaleway/scaleway-sdk-go/pull/128/files#diff-1e71dd53fd89dedc69a5a17bc3efb9a1R114)